spec/models/user_spec.rb

```diff
@@ -345,24 +345,35 @@
     end
 
     describe 'proposals methods' do
-      let(:submitter) { create(:submitter, user: user) }
```

Comment:
How was that working before? where is `submitter` defined? 🤔

Reply:
It is defined in factory file 'event_users.rb'